starter-core/src/test/groovy/io/micronaut/starter/build/dependencies/PomCoordinatesUtils.java

```diff
@@ -35,12 +35,12 @@
 import java.util.Optional;
 import java.util.stream.Stream;
 
-public final class CoordinatesUtils {
-    private static final Logger LOG = LoggerFactory.getLogger(CoordinatesUtils.class);
+public final class PomCoordinatesUtils {
+    private static final Logger LOG = LoggerFactory.getLogger(PomCoordinatesUtils.class);
 
     private static final String NODE_NAME_TEXT = "#text";
 
-    private CoordinatesUtils() {
+    private PomCoordinatesUtils() {
```

Comment:
please, keep the old name to avoid an unnecessary breaking change.

Reply:
Same question here : this is a test, how can it be a breaking change?

Reply:
Forget what I said, it's in `test` because that's where I moved the class.